### PR TITLE
feat(filters): display filter button in data mode

### LINKED ISSUE #823: Apply tag filter to globe view
- [x] also apply the tag filter to the globe view
- [x] indicate in the UI that the current globe view is filtered
- [x] give the option to remove the filter
- [x] do not give the option to set new filters in this view

### PR DESCRIPTION
**Pull request by KatvonRivia**
I added a badge without the number of selected tags. It seemed a bit too small especially with two digits and was very hard to read.

image

Here is the design: https://github.com/ubilabs/esa-climate-from-space/issues/823#issuecomment-760016257

What do you think? 

closes #823

### CODE REVIEW

**Comment by pmast** [comment]
Badge without number looks fine to me.

File: src/scripts/components/main/navigation/navigation.tsx [hunk 1]
```diff
@@ -9,21 +9,27 @@ import {StoryIcon} from '../icons/story-icon';
...
   const dispatch = useDispatch();
   const [showMenu, setShowMenu] = useState(false);
+  const [showTags, setShowTags] = useState(false);
   const selectedLanguage = useSelector(languageSelector);
   const savedLanguage = localStorage.getItem(config.localStorageLanguageKey);
   const [showLanguageBubble, setShowLanguageBubble] = useState<boolean>(
     !savedLanguage
   );
+  const selectedTags = useSelector(selectedTagsSelector);
+  const isFilterActive = selectedTags.length > 0;
```
Comment (pwambach) [inline src/scripts/components/main/navigation/navigation.tsx:32]: Which filter do you mean?

File: src/scripts/components/main/navigation/navigation.tsx [hunk 2]
```diff
@@ -34,6 +40,16 @@ const Navigation: FunctionComponent = () => {
         icon={StoryIcon}
         hideLabelOnMobile
       />
+      {isFilterActive && (
```
Comment (pwambach) [inline src/scripts/components/main/navigation/navigation.tsx:43]: I would suggest to rename `isFilterActive` to `isTagFilterActive` or you could also just say `selectedTags.length && `. This makes it easier to understand when the component should be visible.

File: src/scripts/components/main/navigation/navigation.tsx [hunk 3]
```diff
@@ -65,6 +81,7 @@ const Navigation: FunctionComponent = () => {
           <Menu />
         </Overlay>
       )}
+      {isFilterActive && showTags && <TagsBubble selectedTags={selectedTags} />}
```
Comment (pwambach) [inline src/scripts/components/main/navigation/navigation.tsx:84]: The tag selector is not visible when no tag is selected? How can you select a tag then? Also TagsBubble is not a very specific name, I wasn't sure what this component does when reading it.

**Review by pwambach — CHANGES_REQUESTED** [review]

File: src/scripts/components/main/navigation/navigation.tsx (hunk 3 above)
Comment (KatvonRivia) [reply src/scripts/components/main/navigation/navigation.tsx:84]: The tags are selected in the stories, the user is not supposed to do that in the data view: https://github.com/ubilabs/esa-climate-from-space/issues/823

**Review by pwambach — APPROVED** [review]

**Pull request merged**